scipy/special/_add_newdocs.py

```diff
@@ -13044,8 +13044,38 @@ def add_newdoc(name, doc):
     Notes
     -----
 
+    The log function used in the computation is the natural log.
+
     .. versionadded:: 0.13.0
 
+    Examples
+    --------
+
+    We can use this function to calculate the binary logistic loss also
+    known as the binary cross entropy. This loss function is used for
+    binary classification problems and is defined as:
+
+    .. math::
+        L = 1/n * \\sum_{i=0}^n -(y_i*log(pred_i) + (1-y_i)*log(1-pred_i))
```

Comment:
There is a mismatch between the notations in the formula and the description. e.g. here it says `pred_i` and in the text `y_pred`.